### PR TITLE
follow users

### PR DESCRIPTION
**Pull request by tungblvt**
==> Title: follow users

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/14267

### CODE REVIEW

**Comment by tungblvt** [comment]
ready

File: app/controllers/followers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class FollowersController < ApplicationController
```
Comment (huuphuc93) [inline app/controllers/followers_controller.rb]: k dùng thì xóa đi nhé

File: app/models/user.rb [hunk 2]
```diff
@@ -35,4 +41,21 @@ def from_omniauth auth
       end
     end
   end
+
+  def following? current_user_id, user_id
+    is_followed = Follower.find_by(follower_id: current_user_id, followed_id: user_id)
```
Comment (huuphuc93) [inline app/models/user.rb]:
```suggestion
    Follower.find_by(follower_id: current_user_id, followed_id: user_id).blank?
```

**Comment by tungblvt** [comment]
ready

**Comment by tungblvt** [comment]
ready

File: app/controllers/home_controller.rb [hunk 3]
```diff
@@ -1,5 +1,7 @@
 class HomeController < ApplicationController
   before_action :authenticate_user!
 
-  def index; end
+  def index
+    @supports = Supports::Home.new(current_user, params[:page])
```
Comment (huuphuc93) [inline app/controllers/home_controller.rb]: xóa ()

**Comment by tungblvt** [comment]
ready

**Pull request merged**